ts/features/payments/home/screens/PaymentsHomeScreen.tsx

```diff
@@ -1,3 +1,65 @@
+import { HeaderFirstLevel, VSpacer } from "@pagopa/io-app-design-system";
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { useFocusEffect } from "@react-navigation/native";
+import * as O from "fp-ts/lib/Option";
+import { pipe } from "fp-ts/lib/function";
 import * as React from "react";
+import I18n from "../../../../i18n";
+import { useIODispatch, useIOSelector } from "../../../../store/hooks";
+import { walletPaymentGetUserWallets } from "../../payment/store/actions/networking";
+import { selectWalletPaymentHistoryArchive } from "../../history/store/selectors";
+import { walletPaymentUserWalletsSelector } from "../../payment/store/selectors";
+import PaymentMethodsSection from "../components/PaymentsHomeScreenMethodsSection";
+import PaymentHistorySection from "../components/PaymentsHomeScreenHistorySection";
 
-export const PaymentsHomeScreen = () => <></>;
+export const PaymentsHomeScreen = () => {
+  const dispatch = useIODispatch();
+
+  useFocusEffect(
+    React.useCallback(() => {
+      dispatch(walletPaymentGetUserWallets.request());
+    }, [dispatch])
+  );
+
+  const paymentMethodsPot = useIOSelector(walletPaymentUserWalletsSelector);
+  const paymentHistory = useIOSelector(selectWalletPaymentHistoryArchive) ?? [];
```

Comment:
Attention: this is the payments history which includes the last 10 unfinished, completed and failed payments.
Here instead we need the list of completed transactions, which (I think) comes from the BE.

Reply:
okay, adressed in [d3c3eaa](https://github.com/pagopa/io-app/pull/5535/commits/d3c3eaae811240fc65e9c5c5c3e0e28a12c4ee35) ! 
Do note, when testing, that as of now the first time you open the page you'll see a blank payment history (when running the dev-server, at least) due to a faulty polling/backoff implementation that returns `[error:max-retries]`, it will be fixed in the next PR or directly when we switch to the new apis.